src/engine/test/source/hlp/hlp_test.cpp

```diff
@@ -173,3 +173,249 @@ TEST(map_test, incomplete_map_test)
     ASSERT_TRUE(result2.empty());
     ASSERT_TRUE(result3.empty());
 }
+
+TEST(hlpTimestampTests, ansic)
+{
+    static const char *logQl   = "[<timestamp>]";
```

Comment:
I think we should specify the timestamp format to guarantee that the timestamp isn´t parsed with a different format that still works.

Reply:
done